tasks/main.yml

```diff
@@ -64,6 +71,25 @@
 - name: Restart nginx
   service: name=nginx state=restarted
 
+- name: Copy screen session configuration
+  template:
+    src: "ocim-screenrc"
+    dest: "{{ opencraft_screenrc_path }}"
+    owner: ubuntu
+    group: ubuntu
+
+- name: Copy systemd service file
+  template:
+    src: "ocim-screen.service"
+    dest: "/lib/systemd/system/ocim-screen.service"
```

Comment:
I think this is the location for package provided services, should the service instead be placed in `/etc/systemd/system/`?

Reply:
I thinik you are right, and this is what I first tried for the systemd services in the `ansible-load-balancer` role, but I could not make it work correctly.  So while I might be violating some policy here, this solution has the advantage that it works. :)

Reply:
I did some experiments, and I figured out what I did wrong: I placed the service definition file directly in `/etc/systemd/system/multi-user.target.wants`.  However, this is the directory `systemctl enable` will create the symlink in.  Placing the service definition in the parent directory makes everything work correctly.  The paths are documented in the `systemd.unit(5)` man page, and it's exactly as you said.